vagga.yaml

```diff
@@ -121,6 +121,7 @@ commands:
   test: !Command
     description: Run self tests
     container: test
+    prerequisites: [make]
```

Comment:
Deleted it because it's not related to the topic.
But is `prerequisites` and `-m` the same? There are examples for build as prerequisite for run.